storages/compress.py

```diff
@@ -0,0 +1,14 @@
+from gzip import GzipFile
+
+from storages.utils import GzipCompressionWrapper
```

Comment:
Can you also move the `GzipCompressionWrapper` into this file.

Reply:
Sure, good call